### ISSUE TITLE
Integrating via SPM fails to compile with multiple errors

### ISSUE THREAD

**Issue by khaptonstall** [Appboy/appboy-segment-ios#43]
I'm attempting to integrate using the new SPM integration steps but am encountering a couple build issues, even on a fresh project.

**Steps to Reproduce:** 
1. Create a brand new iOS project (using Storyboards vs SwiftUI)
2. Follow the steps in the v3.6.0 [release notes](https://github.com/Appboy/appboy-segment-ios/releases/tag/3.6.0) to add appboy-segment-ios via SPM (selecting the Full-SDK option)
3. Add [Segment](https://github.com/segmentio/analytics-ios) dependency via SPM (v4.1.2)
5. Build your app

I receive the following build errors:
```
In file included from /Users/.../SourcePackages/checkouts/appboy-segment-ios/Sources/Full-SDK/SEGAppboyHelper.m:2:
/Users/.../SourcePackages/checkouts/appboy-segment-ios/Sources/Full-SDK/SEGAppboyIntegration.h:8:44: error: no type or protocol named 'SEGIntegration'
@interface SEGAppboyIntegration : NSObject<SEGIntegration>
                                           ^
/Users/.../SourcePackages/checkouts/appboy-segment-ios/Sources/Full-SDK/SEGAppboyHelper.m:6:9: fatal error: 'Appboy-iOS-SDK/AppboyKit.h' file not found
#import "Appboy-iOS-SDK/AppboyKit.h"
```

Here's an fresh example project that produces the errors:
[SegmentAppboySPM.zip](https://github.com/Appboy/appboy-segment-ios/files/5668696/SegmentAppboySPM.zip)

**Comment by bdrelling** [comment]
Any word on this? It's also stopping us from migrating a number of dependencies from CocoaPods to SPM.

**Comment by chshapiro** [comment]
Hey @khaptonstall and @bdrelling,

We have been able to repro and are investigating this issue.

**Comment by chshapiro** [comment]
Hey @khaptonstall and @bdrelling,

We just released version 3.6.1 which includes a fix for this issue. If you have any further questions, feel free to reopen the issue.
